scripts/grep-switched.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
this script is an old artefact. please, remove it.

Reply:
Done.